core/src/main/java/org/easymock/internal/Injector.java

```diff
@@ -154,6 +169,32 @@ private static MockType mockTypeFromAnnotation(Mock annotation) {
         return mockType;
     }
 
+    /**
+     * Tries instantiating a class annotated with {@link TestSubject} using its constructor.
+     * @param testSubjectField
+     */
+    private static Object instantiateTestSubject(Field testSubjectField) throws ReflectiveOperationException {
+        Class<?> type = testSubjectField.getType();
+        try {
+            Constructor<?> defaultConstructor = type.getDeclaredConstructor();
+            defaultConstructor.setAccessible(true);
+            return defaultConstructor.newInstance();
+        } catch (NoSuchMethodException e) {
+            System.out.println(e);
+        }
+
+        List<Constructor<?>> accessibleConstructors = stream(type.getDeclaredConstructors())
```

Comment:
From here, let's try finding the constructor we should use. To do that, the \@TestSubject has to have only one constructor, or we need some criteria on which one is better than the other (e.g., use the constructor that has the largest number of arguments?).